frontend/src/layouts/pageHeader.ts

```diff
@@ -22,32 +24,66 @@ function navigateBreadcrumb(e: MouseEvent, href: string) {
   el.dispatchEvent(evt);
 }
 
-export function pageBreadcrumbs(breadcrumbs: Breadcrumb[]) {
+export function breadcrumbSeparator() {
+  return html`<span
+    class="font-mono font-thin text-neutral-300"
+    role="separator"
+    >/</span
+  > `;
+}
+
+const separator = breadcrumbSeparator();
+const skeleton = html`<sl-skeleton class="w-48"></sl-skeleton>`;
+
+function breadcrumbLink({ href, content }: Breadcrumb, classNames?: string) {
+  if (!content) return skeleton;
+
   return html`
-    <sl-breadcrumb class="part-[base]:h-6">
+    <a
+      class=${clsx(
+        tw`flex h-5 items-center gap-1 truncate whitespace-nowrap leading-5`,
+        href
+          ? tw`font-medium text-neutral-400 transition-colors hover:text-neutral-500`
```

Comment:
`text-neutral-400` (#a1a1aa) on a white background has pretty low contrast, definitely below what WCAG asks for. I checked because I also found it a little difficult to read - maybe we could bump the base and on-hover values up another level?

Reply:
[APCA results for this colour combo](https://www.myndex.com/APCA/?BG=ffffff&TXT=a1a1aa&DEV=G4g&BUF=A22)

Reply:
Agree with tessa here, could we bump this up to 500 with the current color scales we're using?

Reply:
Gonna also suggest 700 on hover, it's nice to have a little more contrast for different states